src/client/auth/auth.proto

```diff
@@ -63,7 +62,7 @@ message ACL {
 // Authorization API
 
 message AuthorizeRequest {
-    pfs.Repo repo = 1;
+    string repo = 1;
```

Comment:
Per our discussion, it's to avoid a dependency cycle between pfs.proto and auth.proto. Feel free to ignore this.